.github/workflows/rust.yml

```diff
@@ -19,13 +19,13 @@ jobs:
     - name: Check
       run: cargo check
 
-  clippy:
+  werror:
```

Comment:
wondering why the name change here as i don't know if it has any specific meaning in the workflow file